pkg/policies/flowcontrol/actuators/workload-scheduler/scheduler.go

```diff
@@ -520,13 +523,23 @@ func SanitizeSchedulerProto(proto *policylangv1.Scheduler) *policylangv1.Schedul
 		proto.DefaultWorkloadParameters = p
 	}
 
+	workloadNames := make(map[string]bool)
+
 	// Loop through the workloads
-	for _, workloadProto := range proto.Workloads {
+	for workloadIndex, workloadProto := range proto.Workloads {
+		if workloadProto.Name != "" {
+			if workloadNames[workloadProto.Name] {
```

Comment:
Should add str(index) and default in workloadNames to check for all clashes

Reply:
[556ff51](https://github.com/fluxninja/aperture/pull/2045/commits/556ff51fcd3f6ede4b336358952960cefdf394e3) addresses this.